spec/rest-api/ddl-service.yaml

```diff
@@ -0,0 +1,500 @@
+openapi: 3.0.0
```

Comment:
Can we add a REST API to service this file ? Something like `dae/v1/apis/spec` ? For exact API path, could you please dig out Gang's REST API guide ?

Reply:
Will add in another PR AND a ticket is created.